science/volk/Portfile

```diff
@@ -126,7 +122,7 @@ configure.cppflags-delete -I${prefix}/include
 # specify the Python dependencies; these are checked for at configure,
 # then used for building, but not at runtime.
 
-set pythons_suffixes {35 36 37 38 39 310}
+set pythons_suffixes {35 36 37 38 39 310 311 312}
```

Comment:
can we remove all the EOL Python versions? You'd need to double check that there are no dependents, of course.

Reply:
I have no idea which ones are EOL, however it also seems a bit pointless to not add them when the Python versions are in Macports.

ie that seems like a policy decision outside the scope of a single port.

Reply:
well, you can easily find that right... Everything below 3.9 is EOL and it's a MacPort policy - that's why I am asking. From the variant description it appears Python is only a build dependency and thus it *should* be safe to do so. But please verify that it doesn't install Python bindings or something like that, and see if other ports depend on a specific variant (unlikely). 

Also, your commit message doesn't follow our guidelines. It should be "portname: short description".